css/style.css

```diff
@@ -44,35 +44,113 @@ body {
 }
 
 .bottom-bar {
-  font-family: 'Courier New', Courier, monospace;
   position: absolute;
-  background: #white;
   left: 0;
   bottom: 0;
   height: 50px;
   width: 100%;
-/*  overflow: hidden;*/
 }
 
-.bottom-bar ul {
-  margin: 0% 25% 0 25%;  
+/* helpers/accessibility.css */
+
+.invisible {
+  left: -999px;
+  overflow: hidden;
+  position: absolute;
+  top: -999px;
+}
+
+/* helpers/align.css */
+
+.align {
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+/* modules/anchor.css */
+
+a {
+  outline: 0;
 }
 
-.bottom-bar ul li {
-  border-left: 2px solid white;
-  border-right: 2px solid white;
+/* modules/icon.css */
+
+.icons {
+  display: none;
+}
+
+.icon {
   display: inline-block;
-  margin: 5px;
+  fill: #97adc6;
+  font-size: 16px;
+  font-size: 1rem;
+  height: 1em;
+  vertical-align: middle;
+  width: 1em;
+}
+
+.icon--2x {
+  font-size: 32px;
+  font-size: 2rem;
+}
+
+/* modules/image.css */
+
+svg {
+  max-width: 100%;
 }
 
-.bottom-bar ul li a{
-  margin: 5px;
-  color: rgb(185, 175, 175);
+/* modules/navigation.css */
+
+.navigation {
+  background-color: #f0f6ff;
```

Comment:
As @nityanandagohain said, It is looking a bit absurd upon the current color scheme. Can you change the color please?
Something like : `rgba(255,255,255,0.4)` Change the transparency according to need to get a beautiful looking bottombar